pyseir/cli.py

```diff
@@ -60,24 +61,42 @@ def _generate_state_reports(state=None):
         report.generate_report()
     else:
         for state in us.states.STATES:
-            _generate_state_reports(state.title())
+            _generate_state_reports(state.name.title())
+
+
+def _map_outputs(state=None, output_interval_days=4):
+    output_interval_days = int(output_interval_days)
+    if state:
+        web_ui_mapper = WebUIDataAdaptorV1(state, output_interval_days=output_interval_days)
+        web_ui_mapper.generate_state()
+    else:
+        for state in us.states.STATES:
+            _map_outputs(state.name.title(), output_interval_days)
 
 
-def _run_all(state=None):
+def _run_all(state=None, run_mode='default', generate_reports=True, output_interval_days=4, skip_download=False):
     exceptions = []
-    cache_all_data()
+
+    if not skip_download:
+        cache_all_data()
 
     if state:
         _impute_start_dates(state.title())
```

Comment:
What is difference between state.title and state.name.title? Should this be state.name.title or the state.name.title should be updated to this? This comment applies throughout this PR.

Reply:
Clarified. One was an object the other is a string.